<ui version="4.0" >
 <class>ModelDialog</class>
 <widget class="QWidget" name="ModelDialog" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Modelo de Velocidad</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>&amp;Distancia Maxima</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string> Numero de Muestras</string>
        </property>
        <property name="buddy" >
         <cstring>xsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbN1" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>11</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Distancia</string>
        </property>
        <property name="buddy" >
         <cstring>zsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="sbD1" >
        <property name="maximum" >
         <number>50</number>
        </property>
        <property name="singleStep" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>&amp;Profundidad Maxima</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Numero de Muestras</string>
        </property>
        <property name="buddy" >
         <cstring>xsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbN2" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>13</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Profundidad</string>
        </property>
        <property name="buddy" >
         <cstring>zsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="sbD2" >
        <property name="maximum" >
         <number>50</number>
        </property>
        <property name="singleStep" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>&amp;Velocidades</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Capa 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="sbVel1" >
          <property name="minimum" >
           <number>100</number>
          </property>
          <property name="maximum" >
           <number>100000</number>
          </property>
          <property name="singleStep" >
           <number>100</number>
          </property>
          <property name="value" >
           <number>2500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Capa 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="sbVel2" >
          <property name="minimum" >
           <number>100</number>
          </property>
          <property name="maximum" >
           <number>100000</number>
          </property>
          <property name="singleStep" >
           <number>100</number>
          </property>
          <property name="value" >
           <number>3000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Capa 3:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="sbVel3" >
          <property name="minimum" >
           <number>100</number>
          </property>
          <property name="maximum" >
           <number>100000</number>
          </property>
          <property name="singleStep" >
           <number>100</number>
          </property>
          <property name="value" >
           <number>3300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>&amp;Opciones</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbColor" >
          <item>
           <property name="text" >
            <string>Color</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Grises</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkLeyenda" >
          <property name="text" >
           <string>Ver &amp;Leyenda</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Titulo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leTitulo" >
          <property name="text" >
           <string>Perfil de Velocidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
